packages/embed/src/css/sidetab.scss

```diff
@@ -37,9 +37,11 @@
   &-button {
     position: absolute;
     top: 50%;
-    transform: rotate(-90deg);
-    transform-origin: bottom left;
-    width: 200px;
+    left: -48px;
+    transform: rotate(-90deg) translateX(-50%);
+    transform-origin: left top;
+    min-width: 100px;
```

Comment:
This is nice!
<img src="https://user-images.githubusercontent.com/1561771/144019247-126a9754-2573-4b19-87b4-d35a40e2d8a8.png" width=360 />